### PR TITLE
Add MissingSidecar tooltip

### PR DESCRIPTION
**Pull request by lucasponce**
Fixes https://github.com/kiali/kiali/issues/3535

image

### CODE REVIEW

File: src/components/MissingSidecar/MissingSidecar.tsx [hunk 1]
```diff
@@ -17,7 +19,8 @@ type MissingSidecarProps = {
 class MissingSidecar extends React.Component<MissingSidecarProps, {}> {
   static defaultProps = {
     text: 'Missing Sidecar',
-    textTooltip: 'Missing Sidecar',
+    textTooltip:
+      "Istio container doesn't found in Pod(s). Check if the istio-injection label/annotation is correctly set on the namespace/workload.",
```
Comment (jshaughn) [inline src/components/MissingSidecar/MissingSidecar.tsx:23]: grammar --> "Istio container not found in Pod(s)."    Or perhaps "Istio sidecar container not found in Pod(s)."

File: src/pages/ServiceDetails/ServiceInfo/__tests__/__snapshots__/ServiceInfoDescription.test.tsx.snap [hunk 2]
```diff
@@ -69,7 +69,7 @@ exports[`#ServiceInfoDescription render correctly with data should render servic
                     icon={[Function]}
                     namespace="my-namespace"
                     text="Missing Sidecar"
-                    textTooltip="Missing Sidecar"
+                    textTooltip="Istio container doesn't found in Pod(s). Check if the istio-injection label/annotation is correctly set on the namespace/workload."
```
Comment (jshaughn) [inline src/pages/ServiceDetails/ServiceInfo/__tests__/__snapshots__/ServiceInfoDescription.test.tsx.snap:72]: ditto

**Review by jshaughn — CHANGES_REQUESTED** [review]

**Comment by lucasponce** [comment]
@hhovsepy @jshaughn what is missing in this PR ?

**Review by hhovsepy — APPROVED** [review]

**Review by jshaughn — APPROVED** [review]
Nothing missing, sorry, fell off the radar.

**Pull request merged**